uploader/uploader.py

```diff
@@ -40,7 +40,15 @@ def __init__(self):
                 raise RuntimeError('Won\'t let you upload to localhost because I '
                                    'remove files after uploading them, and you '
                                    'might be uploading to the same directory')
+        url_check = re.search(r'\/$', self.url)
```

Comment:
`str.endswith` (https://docs.python.org/3/library/stdtypes.html#str.endswith) can also be used and reads easier:

```python
if not self.url.endswith('/'):
    # etc
```

Reply:
I tried that and it had a load of issues with it.

If you want me to persevere I will look in the coming days

Reply:
What were the issues? As far as I can tell, it says the same thing.